rocky/tests/test_api.py

```diff
@@ -11,3 +11,9 @@ def test_api_2fa_enabled(client, settings, admin_user):
 
     response = client.get("/api/v1/organization/", headers={"Authorization": f"Token {token}"})
     assert response.status_code == 200
+
+
+# Regression test for https://github.com/minvws/nl-kat-coordination/issues/3754
+def test_auth_header_wrong_format(client, settings, admin_user):
+    response = client.get("/api/v1/organization/", headers={"Authorization": "Not a token"})
+    assert response.status_code == 401
```

Comment:
I suggest also (partially) checking the error message/ response. During QA I noticed variations in messages for this scenario compared to cases where for example an incorrect token is passed

Reply:
What the goal of this test is that we don't regress and make sure that this code path keeps doing the right thing and will return a 401. We could also test the message, but I think it's not really that important to spend time on.